composer.json

```diff
@@ -55,7 +55,8 @@
         "psr/cache-implementation": "To use metadata caching.",
         "symfony/cache": "To have metadata caching when using Symfony integration.",
         "symfony/config": "To load XML configuration files.",
-        "symfony/twig-bundle": "To use the Swagger UI integration."
+        "symfony/twig-bundle": "To use the Swagger UI integration.",
+        "swagger-api/swagger-ui": "^2.2"
```

Comment:
So if assets are not installed an exception should be thrown with a link to the documentation which explains how to install Swagger UI.
Another solution is to remove the HTML format from the default formats but it is less intuitive for users.

cc @dunglas